x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -141,7 +141,23 @@ export class MBMapContainer extends React.Component {
 
   _getFeaturesUnderPointer(mbLngLatPoint) {
     const mbLayerIds = this._getMbLayerIdsForTooltips();
-    return this._mbMap.queryRenderedFeatures(mbLngLatPoint, { layers: mbLayerIds });
+    const PADDING = 2;//in pixels
+    const mbBbox = [
+      {
+        x: mbLngLatPoint.x - PADDING,
+        y: mbLngLatPoint.y - PADDING
+      },
+      {
+        x: mbLngLatPoint.x + PADDING,
+        y: mbLngLatPoint.y + PADDING
+      }
+
+    ];
```

Comment:
Double-check me on this, but if we're padding in the southwest and northeast directions, wouldn't it be:
`
const mbBbox = [
      {
        x: mbLngLatPoint.x - PADDING,
        y: mbLngLatPoint.y + PADDING
      },
      {
        x: mbLngLatPoint.x + PADDING,
        y: mbLngLatPoint.y - PADDING
      }

     ];
`

Reply:
these are pixel coordinates in the view-references system.

Reply:
Chatted a bit about this offline. It's not what I expected from the docs but @thomasneirynck gave me a good overview. No change necessary!